scripts/test.sh

```diff
@@ -632,6 +632,19 @@ function genproto_pass {
   "${ETCD_ROOT_DIR}/scripts/verify_genproto.sh"
 }
 
+function goimport_for_module {
+  GOFILES=$(run go list  --f "{{with \$d:=.}}{{range .GoFiles}}{{\$d.Dir}}/{{.}}{{\"\n\"}}{{end}}{{end}}" ./...)
+  TESTGOFILES=$(run go list  --f "{{with \$d:=.}}{{range .TestGoFiles}}{{\$d.Dir}}/{{.}}{{\"\n\"}}{{end}}{{end}}" ./...)
+  cd "${ETCD_ROOT_DIR}/tools/mod"
+  FILESNEEDSFIX=$(echo "${GOFILES}" "${TESTGOFILES}" | grep -v '.gw.go' | grep -v '.pb.go' | xargs -n 100 go run golang.org/x/tools/cmd/goimports -l -local go.etcd.io)
```

Comment:
Thanks for raising this @HanningLin!

I'm curious about the `-n 100` for xargs, wouldn't that mean the the full output of files needing to be fixed wouldn't be printed if it was more than 100 files?  If so can we just leave that flag off?

Reply:
It should be fine. It means at most 100 files per command line.


```
       -n max-args, --max-args=max-args
              Use at most max-args arguments per command line.  Fewer
              than max-args arguments will be used if the size (see the
              -s option) is exceeded, unless the -x option is given, in
              which case xargs will exit.
```

The script is copied from [fix.sh#L34-L39](https://github.com/etcd-io/etcd/blob/c6d8b65ea7a77556f44a826eecf08c83cc966bca/scripts/fix.sh#L34-L39). The only minor comment is duplication.

Reply:
Ahh right, xargs will split and sequentially send to  `go run` ok in that case this all looks good to me.

Reply:
Yes, this is mostly copied from fix.sh but with a minor diff using `-l` to list the files